spotbugs-tests/src/test/java/edu/umd/cs/findbugs/detect/Issue3280Test.java

```diff
@@ -0,0 +1,14 @@
+package edu.umd.cs.findbugs.detect;
+
+import edu.umd.cs.findbugs.AbstractIntegrationTest;
+import org.junit.jupiter.api.Test;
+
+class Issue3280Test extends AbstractIntegrationTest {
+
+    @Test
+    void testIssue() {
+        performAnalysis("ghIssues/Issue3280.class");
+
+        assertNoBugType("SING_SINGLETON_IMPLEMENTS_CLONEABLE");
```

Comment:
You could also add
`assertBugInClass("SING_SINGLETON_GETTER_NOT_SYNCHRONIZED", "Issue3280");`

Reply:
Good point, I have added this check as well!